src/models/mods.rs

```diff
@@ -27,8 +27,12 @@ pub struct Mod {
 
     /// The title or name of the mod.
     pub title: String,
+    ///The URL of the icon of the mod
+    pub icon_url: String,
     /// A short description of the mod.
     pub description: String,
+    /// A long description of the mod.
+    pub body_url: String,
```

Comment:
Why url ? Will this be on the backblaze bucket ?

Reply:
Yeah. Storing it in the DB is inefficient and bloats the size of the database by a ton.